src/components/card/horizontal-resource-card.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react'
+import {
+  Card,
+  CardPreview,
+  CardHeader,
+  CardContent,
+  CardBody,
+  CardMeta,
+} from './index'
+import Image from 'next/image'
+import Link from 'next/link'
+import Markdown from '../markdown'
+import {track} from 'utils/analytics'
+import {get} from 'lodash'
+
+const HorizontalResourceCard: React.FC<any> = ({
+  resource,
+  location,
+  className = 'border-none my-4',
+  ...props
+}) => {
+  className = `${className} flex sm:flex-row flex-col sm:space-x-5 space-x-0 sm:space-y-0 space-y-5 items-center sm:text-left text-center`
+
+  return (
+    <Card {...props} className={className}>
```

Comment:
fundamentally all of this translates to a stack of divs, so the abstraction feels heavy right now, but I think it sets the stage for a robust and flexible card to iterate on.

Still lots of decisions to make. I like the semantics that this structured approach makes in the component hierachy.